lib/mumukit/login/provider/cas.rb

```diff
@@ -0,0 +1,33 @@
+class Mumukit::Login::Provider::Cas < Mumukit::Login::Provider::Base
+  def configure_omniauth!(omniauth)
+    omniauth.provider :cas,
+                      url: cas_config.url,
+                      host: cas_config.host,
+                      disable_ssl_verification: cas_config.disable_ssl_verification,
+                      ca_path: '.'
+  end
+
+  private
+
+  def cas_config
+    Mumukit::Login.config.cas
+  end
+end
+
+# Monkey-patching to support phpCAS implementation
+# where the first time the 'ticket' param is not sent.
+module OmniAuth
+  module Strategies
+    class CAS
+      alias_method :__callback_phase__, :calback_phase
```

Comment:
typo in callback

Reply:
running tests in development failed because of the typo but they didn't fail in the travis build (????) :thinking::thinking::thinking::thinking: